enzyme/Enzyme/Enzyme.cpp

```diff
@@ -681,11 +698,16 @@ class Enzyme : public ModulePass {
     const AugmentedReturn *aug;
     switch (mode) {
     case DerivativeMode::ForwardModeVector:
+      newFunc = Logic.CreateForwardDiff(
+          cast<Function>(fn), retType, constants, TLI, TA,
+          /*should return*/ false, /*dretPtr*/ false, mode, /* width */ 3,
```

Comment:
It would be nice to have this automatically deduced rather than hard coded, or perhaps come as a parameter?

Reply:
Of course this is just my quick and dirty proof of concept

Reply:
It’s mainly a draft of the C calling convention for vector mode

Reply:
Yeah for sure, just noting this for the future.